packages/@okta/vuepress-site/docs/reference/saml-hook/index.md

```diff
@@ -334,14 +334,18 @@ The `value` object is where you specify the specific operation to perform. It is
 
 Specify the location within the assertion where you want to apply your operation using a slash-delimited path, which follows JSON Patch conventions.
 
-When you perform an `add` op to add a new attribute statement, always begin with `/claims/` and follow that with the name of the new attribute that you are adding.
+When you perform an `add` op to add a new attribute statement, begin with `/claims/` and follow that with the name of the new attribute that you are adding.
```

Comment:
Should we add a sentence about the ability to add to the `/authentication` path? Or is this a limited scenario?

Reply:
Yup this is a limited scenario. It only allows `/authentication/sessionLifetime` which is the special case.